src/parsers/manifestjson.js

```diff
@@ -140,6 +140,16 @@ export default class ManifestJSONParser extends JSONParser {
         this.isValid = false;
       }
     }
+
+    if (!this.parsedJSON.default_locale && this.io) {
+      for (let key in this.io.files) {
+        if (key.startsWith('_locales')) {
```

Comment:
Like you said I'm not a huge fan of the loop but our IO class doesn't yet have a usable `checkIfPathExists` type method. I've filed #1079 for it, though it's obviously low-priority.